CHANGELOG.md

```diff
@@ -7,6 +7,63 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.42.0] - UNRELEASED
+
+### Breaking changes
+
+-   Removed `ThirdPartyEmailPassword` and `ThirdPartyPasswordless` recipes.
+-   `EmailPassword` recipe:
+
+    -   Removed embeddable components: `SignInUp`, `SignInAndUpTheme`
+    -   Removed overridable components: `EmailPasswordSignIn`, `EmailPasswordSignInFooter`, `EmailPasswordSignInHeader`, `EmailPasswordSignUp`, `EmailPasswordSignUpFooter`, `EmailPasswordSignUpHeader`
+    -   Moved `useShadowDom`, `defaultToSignUp`, `privacyPolicyLink`, `termsOfServiceLink` from the config to the root configuration passed to `SuperTokens.init`.
+
+-   `Passwordless` recipe:
+
+    -   Removed embeddable components: `SignInUp`
+    -   Removed overrideable components: `PasswordlessSignInUpHeader_Override`, `PasswordlessSignInUpFooter_Override`
+    -   Removed the `guessInternationPhoneNumberFromInputPhoneNumber` configurable callback, since now the user sets if they are entering email or a phone number explicitly
+    -   Added `defaultToEmail` configuration option, which decides if the contact info input form starts in the email or phone state if the `contactMethod` is set to `EMAIL_OR_PHONE`
+    -   Moved `useShadowDom`, `privacyPolicyLink`, `termsOfServiceLink` from the config to the root configuration passed to `SuperTokens.init`.
+
+-   `ThirdParty` recipe:
+
+    -   Removed embeddable components: `SignInAndUp`
+    -   Removed overridable components: `ThirdPartySignInAndUpHeader`, `ThirdPartySignUpFooter`
+
```

Comment:
the type of prop `components` in AuthRecipeComponentsOverrideContextProvider is incorrectly typed. It requires the dev to pass all components in the map instead of just the one they want to override.